datahub-web-react/src/app/identity/ManageIdentities.tsx

```diff
@@ -23,11 +28,15 @@ const PageTitle = styled(Typography.Title)`
 `;
 
 const Content = styled.div`
+    height: 100%;
```

Comment:
can we use flex: 1 instead?

Reply:
I have kept height because if there is lots of data then container can take only 100% height but yes flex doing the same thing so we can remove height.